content/docs/command-reference/plots/index.md

```diff
@@ -37,18 +37,22 @@ This type of metrics files are created by users, or generated by user data
 processing code, and can be defined in `dvc.yaml` (`plots` field) for tracking
 (optional).
 
-DVC generates plots as HTML files that can be open with a web browser. These
-HTML files use [Vega-Lite](https://vega.github.io/vega-lite/). Vega-Lite is a
-declarative grammar for defining plots using JSON. The plots can also be saved
-as SVG or PNG image filed from the browser.
+DVC is able to handle two general types of plots:
 
-In contrast to `dvc metrics`, these metrics should be stored as data series.
-Unlike its `dvc metrics` counterpart, `dvc plots diff` cannot calculate numeric
-differences between the metrics in different experiments.
+1. Data series files, which can be JSON, YAML, CSV or TSV files.
+2. Image files, like JPGE, GIF, PNG.
+
+DVC generates plots as static HTML websites that can be open with a web browser.
+Data series plots utilize [Vega-Lite](https://vega.github.io/vega-lite/) for
+rendering. Vega-Lite is a declarative grammar for defining plots using JSON. The
+plots can also be saved as SVG or PNG image filed from the browser. Image type
+plots are rendered using `<img>` tag.
```

Comment:
out of curiosity - how do we deal with an image file from a past commit? when I do `dvc plots diff`? do we embed it into HTML, copy it nearby?

Reply:
Currently we just insert files with identical path into same `<div>` tag.:

<img width="512" alt="Screenshot 2021-09-21 at 22 44 56" src="https://user-images.githubusercontent.com/10195618/134244961-108df9f7-f312-4361-b1ad-0eba0a6b9a12.png">


During iterative/dvc#6603 I will try to make it look less terrible.

Reply:
yep, but where does it take the image itself from HEAD? the `png` file - how does it read from the Git history?

Reply:
We read it as a binary file and later write the same way into the webpage.

It can become problematic (memory-wise) in case of multiple images and revision, so further development assumes 
using cache URL instead of binary data (in cached plots use case). In the case of git, well this approach will probably have to remain.

Reply:
@shcheklein Not sure if you meant how images are collected or how they are sourced into the HTML. If you meant the latter, `dvc plots diff` now generates a directory that looks like:

```
dvc_plots
├── index.html
└── static
    ├── HEAD_image.png
    └── workspace_image.png
```

Reply:
Thanks @dberenbaum ! I'm not sure if then `We read it as a binary file` by Pawel is still relevant?

I was asking this because we'll need access to these data from the VS Code env. We are iterating right now on the requirements and I'm curious what kind of interface we can expect/build for it.

Reply:
I think the comment from @pared is relevant for Studio, but probably not for VSCode. We don't have any `--show-json` or other machine-readable output from `dvc plots diff` today for images, so either VSCode can read directly from the generated `dvc_plots` directory, or we will need to add some other option to provide the needed info.

Reply:
I think it's fine for us to start with `dvc plots diff` if it supports experiments/checkpoints. But it doesn't sound like a very efficient thing. Ideally yes, we'll need an interface to see the list of available things across commits/experiments and either path to those or a command that would stream them for us (if it's a Git history) and/or find another way.

Reply:
cc @rogermparent ^^